packages/odyssey-react-mui/src/labs/SideNav/SideNavItemContent.tsx

```diff
@@ -114,6 +92,61 @@ const SideNavItemContent = ({
    */
   scrollRef?: React.RefObject<ScrollIntoViewHandle>;
 }) => {
+  const sidenavItemContentContext = useSideNavItemContent();
+  const contextValue = useMemo(
+    () => sidenavItemContentContext,
+    [sidenavItemContentContext],
+  );
+  const odysseyDesignTokens = useOdysseyDesignTokens();
+  const navItemContentStyles = useMemo(
+    () => ({
+      display: "flex",
+      alignItems: "center",
+      width: "100%",
+      textDecoration: "none",
+      color: `${isDisabled ? odysseyDesignTokens.TypographyColorDisabled : odysseyDesignTokens.TypographyColorHeading} !important`,
+      minHeight: contextValue.isCompact
+        ? odysseyDesignTokens.Spacing6
+        : odysseyDesignTokens.Spacing7,
+      padding: contextValue.isCompact
+        ? `${odysseyDesignTokens.Spacing0} ${odysseyDesignTokens.Spacing4} ${odysseyDesignTokens.Spacing0} calc(${odysseyDesignTokens.Spacing4} * ${contextValue.depth})`
+        : `${odysseyDesignTokens.Spacing2} ${odysseyDesignTokens.Spacing4} ${odysseyDesignTokens.Spacing2} calc(${odysseyDesignTokens.Spacing4} * ${contextValue.depth})`,
+      "&:focus-visible": {
+        borderRadius: 0,
+        outlineColor: odysseyDesignTokens.FocusOutlineColorPrimary,
+        outlineStyle: odysseyDesignTokens.FocusOutlineStyle,
+        outlineWidth: odysseyDesignTokens.FocusOutlineWidthMain,
+        textDecoration: "none",
+        outlineOffset: 0,
+        color: isDisabled
+          ? "default"
+          : `${odysseyDesignTokens.TypographyColorAction} !important`,
+        backgroundColor: !isDisabled
+          ? odysseyDesignTokens.HueNeutral50
+          : "inherit",
+      },
+      "&:hover": {
+        textDecoration: "none",
+        cursor: isDisabled ? "default" : "pointer",
+        color: isDisabled
+          ? "default"
+          : `${odysseyDesignTokens.TypographyColorAction} !important`,
+        backgroundColor: !isDisabled
+          ? odysseyDesignTokens.HueNeutral50
+          : "inherit",
+      },
+    }),
+    [odysseyDesignTokens, contextValue, isDisabled],
+  );
+
+  const NavItemContentContainer = styled("div")(() => ({
+    ...navItemContentStyles,
+  }));
+
+  const NavItemLinkContainer = styled(NavItemLink)(() => ({
+    ...navItemContentStyles,
+  }));
```

Comment:
I hope I didn't mark this wrong in an earlier comment. If so, I apologize. These styles and the creation of `NavItemContentContainer` can be statically assigned above the component, so you won't need `useMemo`.

But I see what we want these two elements to share these styles. I think we need to rethink how we're handling this. I really don't want to be creating new styled elements each render. That will cause performance issues as people open and close the accordion items.

I'm wondering how if simply swapping from Styled Components to Emotion's `css` prop would be better; although, the `css` prop doesn't work correctly in Odyssey last I checked. At that point, we'd need to use `sx` on `Box`. That might make this particular set of styles simpler.

Also, we need to make sure these styles are typed correctly. Currently, they don't have a type. It's `CSSSelector` or something if I remember correctly.